azure/build-image.md

```diff
@@ -88,11 +112,13 @@ export PKR_VAR_offer=0001-com-ubuntu-confidential-vm-jammy
 export PKR_VAR_sku=22_04-lts-cvm
 
 export AA_KBC="cc_kbc_az_snp_vtpm"
+export LIBC=gnu
 export CLOUD_PROVIDER=azure
 PODVM_DISTRO=ubuntu make image
 ```
 
-> **Note**: If you want to disable cloud config then `export DISABLE_CLOUD_CONFIG=true` before building the image.
+> [!NOTE]
+> If you want to disable cloud config then `export DISABLE_CLOUD_CONFIG=true` before building the image.
```

Comment:
it would be more accurate to say "if you want to disable cloud-init", sadly the parameter is still a misnomer

Reply:
Updated.